documents/PropBetPredictProposal.md

```diff
@@ -0,0 +1,145 @@
+## Proposal of MVP version of Bot Blitzer
+
+### Background
+
+We are likely to lose motivation on this project.  I think the best way for this project to stay interesting, is if we can use some form of what we are building almost immediately.  The purpose of this doc is to describe the MVP version of Bot Blitzer which we can start playing immediately, competing in algorithmic fantasy sports play.
+
+### Approach
+
+We focus on NBA because the NBA has games daily until June.  
+
+Every day, each player is asked to respond to 1 or more prop bets of this format
+```json
+{
+    "type": "over",
+    "player": "1",
+    "points": "20",
+    "odds": {
+        "denominator": 4,
+        "numerator": 270,
+    },
+}
+```
+And they are asked to response with whether they'd like to take the best
+```json 
+{
+    "reason": "", \\ displayed but otherwise unused
+    "response": "YES", \\ one of YES,NO,SKIP
+}
```

Comment:
```suggestion
{
    "reason": "", \\ displayed but otherwise unused
    "quantity": -4.0,  \\ zero indicates a "skip", negative indicates a "No" bet of a specified magnitude, positive indicates a "Yes" bet; invalid if wagering an amount outside of the user's current wallet
}
```